clients/src/main/java/org/apache/kafka/common/requests/FindCoordinatorRequest.java

```diff
@@ -70,7 +75,7 @@ public AbstractResponse getErrorResponse(int throttleTimeMs, Throwable e) {
             response.setThrottleTimeMs(throttleTimeMs);
         }
         Errors error = Errors.forException(e);
-        return FindCoordinatorResponse.prepareResponse(error, Node.noNode());
+        return FindCoordinatorResponse.prepareOldResponse(error, Node.noNode());
```

Comment:
Why do we call the `prepareOldResponse` in `getErrorResponse`? Should we be doing a version check here?